tools/ci_build/build.py

```diff
@@ -317,6 +319,14 @@ def generate_build_tree(cmake_path, source_dir, build_dir, cuda_home, cudnn_home
                  "-Donnxruntime_USE_MKLDNN=" + ("ON" if args.use_mkldnn else "OFF"),
                  "-Donnxruntime_USE_MKLML=" + ("ON" if args.use_mklml else "OFF"),
                  "-Donnxruntime_USE_NGRAPH=" + ("ON" if args.use_ngraph else "OFF"),
+                 "-Donnxruntime_USE_OPENVINO=" + ("ON" if args.use_openvino else "OFF"),
+                 "-Donnxruntime_USE_OPENVINO_BINARY=" + ("ON" if args.use_openvino == "MYRIAD_FP16" else "ON"),
+                 "-Donnxruntime_USE_OPENVINO_SOURCE=" + ("OFF" if (args.use_openvino and args.use_openvino != "MYRIAD_FP16") else "OFF"),
```

Comment:
shouldn't onnxruntime_USE_OPENVINO_SOURCE be ON in the ! MYRIAD_FP16 case?

Ran into build failure because of this when trying to build with CPU_FP32 from source.

Reply:
No longer relevant.